framework/tlvf/AutoGenerated/include/tlvf/ieee_1905_1/eLinkMetricNeighborType.h

```diff
@@ -0,0 +1,30 @@
+///////////////////////////////////////
+// AUTO GENERATED FILE - DO NOT EDIT //
+///////////////////////////////////////
+
```

Comment:
This commit is misplaced as it seems - the previous commit did not change tlvf.py..

Reply:
This is not the first time I see this. Looks like github is messing the commits but they are actually ok. 
I've cloned the repository in another directory and double-checked that the order of these commits is correct.

Reply:
@morantr @CoralMalachi I think you found a way to solve this in github?